### PR TITLE
[incubator/cassandra] Add image pull secrets to statefulset template

### PR DESCRIPTION
**Pull request by alebabai**
* update statefulset template
* add new parameter to README
* bump chart version to 0.1.7

### CODE REVIEW

**Comment by KongZ** [comment]
cc @mattfarina @prydonius

File: incubator/cassandra/templates/cassandra-statefulset.yaml [hunk 1]
```diff
@@ -103,6 +103,10 @@ spec:
         volumeMounts:
         - name: data
           mountPath: /var/lib/cassandra
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+        - name: {{ .Values.image.pullSecrets }}
```
Comment (mattfarina) [inline incubator/cassandra/templates/cassandra-statefulset.yaml:108]: `image.pullSecrets` should be in the `values.yaml` file, commented out, and documented.

**Review by mattfarina — CHANGES_REQUESTED** [review]
On the whole I like the idea of having this. I should discuss it with the other maintainers. Just one nit.

File: incubator/cassandra/templates/cassandra-statefulset.yaml (hunk 1 above)
Comment (alebabai) [reply incubator/cassandra/templates/cassandra-statefulset.yaml:108]: done

**Pull request merged**